l10n/en_GB.json

```diff
@@ -43,7 +43,6 @@
     "Unspecified" : "Unspecified",
     "Windows domain qualified name" : "Windows domain qualified name",
     "X509 subject name" : "X509 subject name",
-    "Use SAML auth for the %s desktop clients (requires user re-authentication)" : "Use SAML auth for the %s desktop clients (requires user re-authentication)",
```

Comment:
files within l10n/ will be created and updated automatically, you do not need to care about them and best leave them out from commits/PRs